tdrs-frontend/src/components/Header/Header.jsx

```diff
@@ -92,20 +109,20 @@ function HeaderComp() {
                   )}
                 </li>
                 <li className="usa-nav__secondary-item">
-                  <a
-                    className="sign-out-link"
-                    href={
-                      user && user.email
-                        ? `${process.env.REACT_APP_BACKEND_URL}/logout/oidc`
-                        : `${process.env.REACT_APP_BACKEND_URL}/login/oidc`
-                    }
-                  >
-                    <FontAwesomeIcon
-                      className="margin-right-1"
-                      icon={faSignOutAlt}
-                    />
-                    {user && user.email ? 'Sign Out' : 'Sign In'}
-                  </a>
+                  {!window.location.href.match(
```

Comment:
I wonder if it would be better to invert this so it only runs if the current href contains `localhost`? As is, we'll need to update this check when we get a real production domain.